website/addons/base/__init__.py

```diff
@@ -350,6 +351,46 @@ def deep_url(self):
     def revision(self):
         return getattr(self, '_revision', None)
 
+    def add_tag(self, tag, auth, node, file_name, save=True):
```

Comment:
I believe the `save` flag here is unnecessary. It is used on the `Node` object because saving the `Node` can trigger several long-running actions and celery tasks to update external services. As we've not overridden `GuidFile.save`, I can't see that it serves a purpose here.

Ping @sloria for confirmation of this.